src/foam/u2/md/ActionView.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * @license
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+foam.CLASS({
+  package: 'foam.u2.md',
+  name: 'ActionView',
+  extends: 'foam.u2.View',
```

Comment:
It doesn't export 'data'. Or perhaps I should remove the export from View.

On 14 September 2016 at 12:32, Braden Shepherdson <notifications@github.com>
wrote:

> In src/foam/u2/md/ActionView.js
> <https://github.com/foam-framework/foam2-experimental/pull/101#discussion_r78785073>
> :
>
> > + * Licensed under the Apache License, Version 2.0 (the "License");
> > + * you may not use this file except in compliance with the License.
> > + * You may obtain a copy of the License at
> > + *
> > + *     http://www.apache.org/licenses/LICENSE-2.0
> > + *
> > + * Unless required by applicable law or agreed to in writing, software
> > + * distributed under the License is distributed on an "AS IS" BASIS,
> > + * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
> > + * See the License for the specific language governing permissions and
> > + * limitations under the License.
> > + */
> > +foam.CLASS({
> > +  package: 'foam.u2.md',
> > +  name: 'ActionView',
> > +  extends: 'foam.u2.View',
>
> Why? Isn't this a textbook example of a View?
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub
> <https://github.com/foam-framework/foam2-experimental/pull/101/files/16649f82ddf2918828782c00f735d10a13350f61#r78785073>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AHHXs-DB0xGCcCvpxr--EdSrKvjNlc-5ks5qqCG7gaJpZM4J8xKR>
> .
>